# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/style/moz.build
# Commit: aa5527055ad1
# Full Hash: aa5527055ad1d21aa109bc8a0ceaee3b0a58288c
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2022-06-29 03:44:30
# Regressor Bug: 1771493
# File Overlap Count: 1
# Description:
#   Bug 1771493 - Part 1. Split FontFace(Set) into DOM facing and implementation classes. r=emilio
#   
#   This patch aims to split the implementation details of FontFace(Set)
#   into FontFace(Set)Impl classes. The Impl variants have threadsafe
#   refcounting and are intended to be the basis for worker support for
# ==============================================================================

diff -r e0c50a17b0ff -r aa5527055ad1 layout/style/moz.build
--- a/layout/style/moz.build	Tue Jun 28 15:45:58 2022 +0000
+++ b/layout/style/moz.build	Tue Jun 28 15:58:09 2022 +0000
@@ -140,7 +140,9 @@
     "CSSSupportsRule.h",
     "CSSValue.h",
     "FontFace.h",
+    "FontFaceImpl.h",
     "FontFaceSet.h",
+    "FontFaceSetImpl.h",
     "FontFaceSetIterator.h",
     "MediaList.h",
     "MediaQueryList.h",
@@ -188,7 +190,9 @@
     "DocumentStyleRootIterator.cpp",
     "ErrorReporter.cpp",
     "FontFace.cpp",
+    "FontFaceImpl.cpp",
     "FontFaceSet.cpp",
+    "FontFaceSetImpl.cpp",
     "FontFaceSetIterator.cpp",
     "FontPreloader.cpp",
     "GeckoBindings.cpp",